demo/run_validation.ipynb

```diff
@@ -18,14 +18,26 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": 2,
    "metadata": {
     "ExecuteTime": {
-     "end_time": "2023-10-26T23:54:04.578459Z",
-     "start_time": "2023-10-26T23:54:04.572010Z"
+     "end_time": "2023-12-28T03:27:19.801868Z",
+     "start_time": "2023-12-28T03:27:19.728856Z"
     }
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "ename": "ModuleNotFoundError",
+     "evalue": "No module named 'mercury'",
+     "output_type": "error",
+     "traceback": [
+      "\u001B[0;31m---------------------------------------------------------------------------\u001B[0m",
```

Comment:
It may be at this point from all the changes. I haven't run it, so this is old output. Since I updated a few things, it might work? I don't want to spend too long on it since we're going to be moving away from it.